apps/dash-3d-image-partitioning/setup

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+git submodule init && git submodule update
```

Comment:
I think you don't need this line since you included the files which used to be in the submodule in the PR? (thanks for doing this BTW, it will easier for users I think)